arrow-syntax/src/main/kotlin/arrow/syntax/collections/iterable.kt

```diff
@@ -2,8 +2,8 @@ package arrow.syntax.collections
 
 import arrow.core.Option
 import arrow.core.Some
-import arrow.core.andThen
 import arrow.core.getOrElse
+import arrow.syntax.function.andThen
 
 @Deprecated(message = "`collect` is now part of the FunctorFilter interface and generalized to all filterable data types")
 fun <A : Any, B> Iterable<A>.collect(vararg cases: (A) -> Option<B>): List<B> =
```

Comment:
This has been deprecated for the whole `0.10.x` version, no? I think this can also be removed for `0.11.0`.